get_github_coderabbit_review/coderabbit_fetcher/analyzers/resolution_detector.py

```
@@ -0,0 +1,398 @@
+            r"resolved",
+            r"fixed",
+            r"addressed",
+            r"completed",
+            r"done",
+            r"corrected",
+            r"appropriate",
+            r"good",
+            r"ok",
+            r"fine",
+            # 肯定的なフィードバックパターン
+            r"解決.*?[！!]",
```

Ruff (0.13.2)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)